src/inputs/select.scss

```diff
@@ -41,21 +42,6 @@ $iui-after-border: 5px solid;
       }
     }
 
-    &::after {
-      content: '';
```

Comment:
Are you suggesting displaying the SVG using `::after`?  I was exploring adding an SVG to the end of a hyperlink using `::after`.  I based the code off of [this article](https://christianoliff.com/blog/styling-external-links-with-an-icon-in-css/):
```scss
.iui-anchor {
  @include iui-anchor;

  &.iui-anchor-popout::after {
    content: '';
    width: $iui-icons-small;
    height: $iui-icons-small;
    margin-left: $iui-xs;
    background-position: center;
    background-repeat: no-repeat;
    background-size: contain;
    display: inline-block;
    background-image: url("data:image/svg+xml,%3Csvg viewBox='0 0 16 16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m16 0v5.4l-1.9-2-8.4 8.4-1.5-1.5 8.3-8.4-1.9-1.9m5.4 16v-9h-1v8h-14v-14h8v-1h-9v16z'/%3E%3C/svg%3E");
  }
}
```

But I haven't figured out how to pass our color variables through to the icon.  [This link](https://stackoverflow.com/a/27827090) says it is doable in SCSS but we use those weird `t()` things, so I'm not sure how to do so.
![Screen Shot 2022-01-12 at 12 35 06 PM](https://user-images.githubusercontent.com/849817/149192200-7e7e4dad-2f38-4d26-b637-3a7971fec735.png)

Reply:
You might be able to just use `currentColor` to get the value from the text without messing with the `themed()` mixin.

Reply:
Unfortunately it appears that does not work, it gives me the same results as above.
```scss
background-image: url("data:image/svg+xml,%3Csvg viewBox='0 0 16 16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill='currentColor' d='m16 0v5.4l-1.9-2-8.4 8.4-1.5-1.5 8.3-8.4-1.9-1.9m5.4 16v-9h-1v8h-14v-14h8v-1h-9v16z'/%3E%3C/svg%3E");
```

Reply:
Also just tried applying `color: red` right above the `background-image` CSS within the `::after`, still no luck.

Reply:
Found this article just now & playing around with it: [Coloring SVGs in CSS Background Images](https://codepen.io/noahblon/post/coloring-svgs-in-css-background-images)

Reply:
I've got this working, but [browser support is poor](https://caniuse.com/css-masks).
```scss
@supports (-webkit-mask-image: url("https://unpkg.com/@itwin/itwinui-icons/icons/window-popout.svg") or (mask-image: url("https://unpkg.com/@itwin/itwinui-icons/icons/window-popout.svg"))) {
  -webkit-mask-image: url("https://unpkg.com/@itwin/itwinui-icons/icons/window-popout.svg");
  mask-image: url("https://unpkg.com/@itwin/itwinui-icons/icons/window-popout.svg");
  @include themed {
    background-color: t(iui-color-foreground-primary);
  }
}
```

Reply:
Does this work?

```scss
@include themed {
  content: url("data:image/svg+xml<svg viewBox='0 0 16 16' xmlns='http://www.w3.org/2000/svg'><path fill=#{t(iui-icons-color-actionable)} d='m16 0v5.4l-1.9-2-8.4 8.4-1.5-1.5 8.3-8.4-1.9-1.9m5.4 16v-9h-1v8h-14v-14h8v-1h-9v16z'/ ></svg>");
}
```

I'm guessing it will work in light theme but fail in dark theme because it uses CSS variables.

If there is no other solution, lets just go abandon this idea and move on with keeping the svg in html.

Reply:
> I've got this working, but [browser support is poor](https://caniuse.com/css-masks).

The prefixed version does work fine across Chrome/Firefox/Safari for our use case, right? Maybe we could use it with my solution as fallback?

Reply:
The icon is taking up space but there is no color for light or dark theme.
![Screen Shot 2022-01-12 at 1 30 29 PM](https://user-images.githubusercontent.com/849817/149200528-635eb5f4-ed0f-4d03-91ae-6ce52b610adc.png)

Reply:
I have the below working.  It uses `content` to display the icon in #008BE1, then within the `@supports` we remove the `content` and display the icon instead using our color variable & `mask-image`.
```scss
&::after {
  content: url("data:image/svg+xml,%3Csvg viewBox='0 0 16 16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill='%23008BE1' d='m16 0v5.4l-1.9-2-8.4 8.4-1.5-1.5 8.3-8.4-1.9-1.9m5.4 16v-9h-1v8h-14v-14h8v-1h-9v16z'/%3E%3C/svg%3E");
  display: inline-block;
  width: $iui-icons-small;
  height: $iui-icons-small;
  margin-left: $iui-xs;
  background-position: center;
  background-repeat: no-repeat;
  background-size: contain;

  @supports (
    -webkit-mask-image: url('https://unpkg.com/@itwin/itwinui-icons/icons/window-popout.svg') or (mask-image:
    url('https://unpkg.com/@itwin/itwinui-icons/icons/window-popout.svg'))
  ) {
    content: '';
    mask-image: url('https://unpkg.com/@itwin/itwinui-icons/icons/window-popout.svg');
    mask-image: url('https://unpkg.com/@itwin/itwinui-icons/icons/window-popout.svg');
    mask-repeat: no-repeat;
    mask-repeat: no-repeat;
    background-color: var(--iui-color-foreground-primary);
    height: $iui-icons-small + $iui-xxs;
    vertical-align: middle;
  }
}
```

Reply:
Worth mentioning: If we change Select to use `input` instead of `div`, we won't be able to use a psuedo-element (CSS limitation). Same problem in ComboBox. So I guess this discussion is not productive.